addons/l10n_se/util/luhn.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+class luhn:
```

Comment:
It is not usual to put something in the util directory.  You can do a util abstract orm class in models/ .  It would also be good to clearly indicate what is different from stdnum.luhn (if it is already any different).

Reply:
Yes there is a difference. I can move it to models as a orm class.